extensions/amp-document-recommendations/0.1/amp-document-recommendations.js

```diff
@@ -147,10 +168,14 @@ export class AmpDocumentRecommendations extends AMP.BaseElement {
       titleElement.textContent = next.title;
       articleHolder.appendChild(titleElement);
 
-      this.element.appendChild(articleHolder);
-
-      this.appendDivision_();
+      recommendations.appendChild(articleHolder);
+      recommendations.appendChild(this.createDivider_());
     }
+
+    this.element.appendChild(recommendations);
+    this.positionObserver_.observe(recommendations,
+        PositionObserverFidelity.LOW,
+        position => this.positionUpdate_(from - 1, position));
```

Comment:
What does `from - 1` mean? Could you use an additional variable name for this value, or rename `from` so that it describes the semantics a little better? Is `from` the index of the article in the list of recommended articles?

Reply:
Good point, updated.